streams/order-data.go

```diff
@@ -0,0 +1,35 @@
+package streams
+
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+)
+
+type OrderData struct {
+	Rate   float64
+	Type   string
+	Amount float64
+}
+
+func (od *OrderData) UnmarshalJSON(b []byte) error {
+	var rawStrings map[string]string
```

Comment:
What do you want to do @gronnbeck?
The problem is that the data is shitty and its all strings, we have to convert em.
So I would love an example of what you think need to be done